README.md

```diff
@@ -79,7 +79,7 @@ Third-party A-Frame components (of the entity-component system).
 
 - [aframe-jade-boilerplate](https://github.com/KevinGrandon/aframe-jade-boilerplate)
 - [aframe-role](https://github.com/Popmotion/aframe-role)
-- [D3.js](http://blockbuilder.org/enjalot/1fd196cd99f8d58a56d3)
+- [D3.js](http://blockbuilder.org/enjalot/1fd196cd99f8d58a56d3)  (with [video tutorial](https://www.youtube.com/watch?v=Tb2b5nFmmsM))
```

Comment:
Thanks!

Can you add so it will look like `D3.js - A-Frame with D3.js [Video Tutorial]`?